_contributors-app/sudonitesh.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Nitesh Tosniwal
+description: I am an aspiring full stack web developer.
+picUrl: https://avatars2.githubusercontent.com/u/35661486?s=460&v=4
+skills: |
+  - **Languages:** Javascript, Python, HTML/CSS, C/C++
+  - **Frameworks:** NodeJS, ExpressJS, Jquery
+  - **Databae:** MongoDB, MySQL
+  - **Tools:** git, Github, TravisCI
+
+githubUsername: sudonitesh
+speciality: |
+
+  - Developing Responsive Full Stack Web Apps
+
+  - Open Source
+
+github_url: https://github.com/sudonitesh
+linkedIn_url: https://www.linkedin.com/in/sudonitesh/
+twitter_url: https://twitter.com
+
+work_experience: |
+ - Will update soon. 
+
+project:
+- |
+    - [Text Editor](https://github.com/sudonitesh/text-editor): A hosted text-editor that can be used to share the typed text through url and get html code from typed text
+- |
+   - [Wiki Viewer](https://github.com/sudonitesh/fcc-wikipedia): Wikipedia Viewer.
+
+volunteer_experience: |
+ - Third in team based GenHax hackathon. Made a web-app to digitalise garbage-collection of the country.
```

Comment:
A better statement: "I participated in a team based hackathon "GenHax" where we secured the third position. We made a web-app to help digitalise India's garbage collection."